examples/gno.land/p/demo/ufmt/ufmt.gno

```diff
@@ -48,6 +48,16 @@ func Sprintf(format string, args ...interface{}) string {
 			default:
 				buf += "(unhandled)"
 			}
+		case 't':
```

Comment:
Maybe it's just me, but this form look cleaner and easier to read:

```go
switch v := arg.(type) {
case bool:
    if v {
	    buf += "true"
    } else {
	    buf += "false"
    }
default:
    buf += "(unhandled)"
}
```

What do you think?

Reply:
I also prefer that, but the other option is not an issue for me.

Up to @ltzmaxwell.

Reply:
I like this too, thanks @zivkovicmilos @moul